Assets/GothicVR/Scripts/Manager/SkyManager.cs

```diff
@@ -36,32 +51,79 @@ public void InitSky()
                 CreatePresetState(new SkyState(), (state) => state.PresetDay0())
             });
 
+
             RenderSettings.fog = true;
             RenderSettings.fogMode = FogMode.Linear;
             RenderSettings.ambientMode = AmbientMode.Flat;
-            Interpolate();
+            InitRainState();
+            noSky = true;
+
+            Interpolate(new DateTime());
         }
 
-        private void Interpolate(DateTime _)
+        private void UpdateStateTexAndFog(SkyState currentState)
         {
-            Interpolate();
+            if (SettingsManager.GameSettings.GothicINISettings.ContainsKey("SKY_OUTDOOR"))
+            {
+                var currentDay = GameTime.I.GetDay();
+                var day = (currentDay + 1);
+
+                // hacky way to use the proper color for the current day until animTex is implemented
+                var colorValues =
```

Comment:
Nice, but be careful. If setting is gone from Ini, game is crashing. Please ensure we have a LogError here.